Users/toml_config.rst

```diff
@@ -0,0 +1,155 @@
+Writing a coala Configuration File in TOML
+======================================================
+
+This document describes how to write configuration files for
+coala in TOML format.
+
+Naming, Scope and Location
+--------------------------
+
+You can use up to three coafiles to configure your project.
+
+1. A project-wide coafile.
+2. A user-wide coafile.
+3. A system-wide coafile.
+
+Project-Wide coafile
+~~~~~~~~~~~~~~~~~~~~
+
+It is a convention that the project-wide configuration file is named
+``.coafile.toml`` and lies in the project root directory.
+If you follow this convention, simply executing ``coala -T`` from the
+project root will execute the configuration specified in that file.
+
+Settings given in the project-wide configuration file override all settings
+given by other files and can only be overridden by settings given via the
+command line interface.
+
+User-Wide and System-Wide coafile
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can place a ``.coarc.toml`` file in your home directory to set certain
+user wide settings. Those settings will automatically be taken for all
+projects executed with that user.
+
+All settings specified here override only settings given by the system
+wide configuration file which has the lowest priority. The
+``system_coafile.toml`` must lie in the coala installation directory
+and is valid for everyone using this coala installation.
+
+It can be used to define the type of files you usually don't want to lint,
+like minified files (e.g. ``*.min.js``) and backup files (e.g. ``*.orig``)::
+
+    ignore = [ '**.min.js', '**.orig' ]
+
+Basic TOML concepts
+---------------------------------
+This part describes the basic TOML concepts required to write coala
+configuration files
+
+- TOML is case sensitive. So remember to not have duplicate sections/tables
+  or duplicate keys in same section.
+- key-value pairs are building blocks of a TOML document. Use key-value
+  pairs to specify rules for coala bears.
+- A table is a collection of key-value pairs. Use a table for specifying
+  a coala section.
+
+::
+
+    [cli]
+    bears = 'SpaceConsistencyBear'
+    files = 'src/*.c'
+    use_spaces = true
+
+    [invalidlinks]
+    enabled = false
+    files = [ '**/*.rst', 'README.rst']
+    ignore = 'venv/**'
+    bears = 'InvalidLinkBear'
+
+Here tables ``cli`` and ``invalidlinks`` are coala sections.
```

Comment:
This coafile is in TOML format or in coala format? Also specify what type of format is because from the example i can't be sure and for other examples.